unit-tests/src/test/java/org/eclipse/collections/impl/factory/HashingStrategySetsTest.java

```diff
@@ -57,6 +61,10 @@ public void mutable()
         Verify.assertInstanceOf(MutableSet.class, factory.of(HashingStrategies.defaultStrategy(), 1, 2, 3, 4, 5, 6, 7, 8));
         Assert.assertEquals(UnifiedSet.newSetWith(1, 2, 3, 4, 5, 6, 7, 8), factory.ofAll(HashingStrategies.defaultStrategy(), FastList.newListWith(1, 2, 3, 4, 5, 6, 7, 8)));
         Verify.assertInstanceOf(MutableSet.class, factory.of(HashingStrategies.defaultStrategy(), FastList.newListWith(1, 2, 3, 4, 5, 6, 7, 8)));
+        Assert.assertEquals(UnifiedSet.newSet(1), factory.ofInitialCapacity(HashingStrategies.defaultStrategy(), 1));
+        Verify.assertInstanceOf(MutableSet.class, factory.ofInitialCapacity(HashingStrategies.defaultStrategy(), 1));
+        Assert.assertEquals(UnifiedSet.newSet(3), factory.withInitialCapacity(HashingStrategies.defaultStrategy(), 3));
+        Verify.assertInstanceOf(MutableSet.class, factory.ofInitialCapacity(HashingStrategies.defaultStrategy(), 3));
```

Comment:
There needs to be another test which checks for HashingStrategy. Also, another test which has first or last name hashing strategy. Both of these tests can be combined as one.
This will test that the hashing strategy passed in the method is honored.